script.js

```diff
@@ -1,4 +1,188 @@
+// Create a version of the Beat That dice game,
+// where players create the largest number they can by concatenating random dice roll digits.
+//
+// --- Base Requirements:
+// 1. There are 2 players and players take turns.
+// 2. When a player clicks Submit, the game rolls 2 dice and shows the dice rolls,
+//    for example 3 and 6.
+// 3. The player picks the order of the dice they want. For example,
+//    if they wanted the number 63, they would specify that the 2nd dice goes first.
+//    You can choose how the player specifies dice order.
+// 4. After both players have rolled and chosen dice order,
+//    the player with the higher combined number wins.
+
+// ========== GLOBAL VAR ==========
```

Comment:
Well done in splitting up your code into global variables, helper functions and the main logic! It's an underrated but important skill set for any coding profession.

Reply:
Good job in naming your functions using verbs and nouns for variables 👍